test/dx_mgn_pool.js

```diff
@@ -66,6 +66,27 @@ contract("DxMgnPool", (accounts) => {
   })
 
   describe("participateInAuction()", () => {
+
+    it("non-owner can't ping it", async() => {
```

Comment:
Double negation is hard to understand

```suggestion
    it("only owner can trigger it", async() => {
```